installer/default_platform.conf

```diff
@@ -471,7 +470,6 @@ EOF
     fi
 
 # Add extra linux command line
-extra_cmdline_linux=%%EXTRA_CMDLINE_LINUX%%
```

Comment:
>EXTRA_CMDLINE_LINUX [](http://example.com/codeflow?start=22&length=19)

The 3 lines were added in one commit, it makes sense to remove all 3 lines together.